aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/CreateHandler.java

```diff
@@ -26,7 +26,7 @@
         final ResourceModel model = request.getDesiredResourceState();
 
         try {
-            proxy.injectCredentialsAndInvokeV2(Translator.translateToCreateRequest(model),
+            proxy.injectCredentialsAndInvokeV2(Translator.translateToCreateRequest(model, request.getDesiredResourceTags()),
```

Comment:
Nice. Tag on create is the right way to create tags in a following API call. It can allow customer to control the resource permission with tags.